File: 프로그래머스/unrated/133499. 옹알이 （2）/옹알이 （2）.py
def solution(babbling):
    able = ['aya', 'ye', 'woo', 'ma']
    answer = []
    count = 0
    for v in babbling:
        flag = True
        for a in able:
            if a*2 in v:
                flag = False
                break
        if flag: answer.append(v)
    
    for v in answer:
        v = v.replace('aya','A').replace('ye','Y').replace('woo','W').replace('ma','M')
        v = v.replace('A','').replace('Y','').replace('W','').replace('M','')
        # print(v)
        if v == '': count += 1
        
    return count

File: 프로그래머스/unrated/133499. 옹알이 （2）/옹알이 （2）.js
function solution(babbling) {
    let answer = 0;
    for (let i = 0; i < babbling.length; i++) {
        let b = babbling[i];
        let ableList = ['aya', 'ye', 'woo', 'ma'];
        for (let j = 0; j < ableList.length; j++) {
            let able = ableList[j];
            if (b.indexOf(able.repeat(2)) === -1) {
                b = b.replaceAll(able, ' ');
            }
        }
        if (b.replaceAll(' ', '') === '') {
            answer += 1;
        }
    }
    return answer;
}